radicle-cli/examples/rad-inspect-noauth.md

```diff
@@ -0,0 +1,13 @@
+The `rad inspect` command can be run without being authenticated with radicle:
+
+```
+$ rad self
+✗ Self failed: Could not load radicle profile
+✗ To setup your radicle profile, run `rad auth`.
+
+```
+
+```
+$ rad inspect
+rad:z42hL2jL4XNk6K8oHQaSWfMgCL7ji
+```
```

Comment:
Would this be better as a regular test?  Personally speaking, it may not even be necessary to test this at all.  Having `rad inspect` work without authentication is nice and expected behavior.

Keeping documentation tests as documentation first will help end users (and myself) when learning how to use radicle.

Reply:
Could do yeah.

Reply:
Hmm this is more complicated actually, because I don't currently have a way of running a CLI command without a home directory, so will leave it for later.